src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -1075,6 +1076,28 @@ bool FullyConnected::useSparseWeightsDecompression() {
     return true;
 }
 
+void FullyConnected::fuseDecompressionMultiply(const NodePtr& constData) {
+    fuseDecompressionConstant(constData, decompressionMultiply);
+}
+
+void FullyConnected::fuseDecompressionSubtract(const NodePtr& constData) {
+    fuseDecompressionConstant(constData, decompressionSubtract);
+}
+
+void FullyConnected::fuseDecompressionConstant(const NodePtr& constData, std::vector<float>& decompressionValues) {
+    auto *constInputNode = dynamic_cast<node::Input *>(constData.get());
```

Comment:
in fact the same code block is used in three different places inside the plugin already (node.cpp, rnn.cpp, fullyconnected.cpp).
What about introduction of castToVector method into Input node API (same as cast_vector inside Constant ngraph op)?

Reply:
I agree with the idea. But can we do that in a separate PR? I took a look on ngraph cast_vector realization: it is not trivial, and it may take extra time to do smth the same in CPU Input node

Reply:
Agree. Lets follow-up in separate PR